cli/src/util.rs

```diff
@@ -18,6 +22,33 @@ svg { width: 100%; }
 
 ";
 
+pub fn lang_not_found_for_path(path: &Path, loader_config: &LoaderConfig) -> String {
+    let path = path.display();
+    format!(
+        indoc! {"
+            No language found for path `{}`
+
+            If a language should be associated with this file extension, please ensure the path to `{}` is inside one of the following directories as specified by your 'config.json':\n\n{}\n
```

Comment:
I think this is incorrect for `tree-sitter highlight`. I built 2d652c90a7c60efe4b2820e257a7e79c1c5017d9 and it prints e.g.

```
$ tree-sitter highlight foo.bar
No language found for path `foo.bar`

If a language should be associated with this file extension, please ensure the path to `foo.bar` is inside one of the following directories as specified by your 'config.json':

  1. /home/ruud/repos
```

The file I’m trying to highlight should not need to be specified in the `config.json`.

Reply:
oh sorry, it should be extension of `foo.bar`, there must be a grammar dir where the filetype of `foo.bar` (bar) is in the repo's package.json